addons/website/static/src/js/editor/snippets.options.js

```diff
@@ -367,29 +322,29 @@ const FontFamilyPickerUserValueWidget = SelectUserValueWidget.extend({
                     return;
                 }
                 if (googleFontServe) {
-                    this.googleFonts.push(font);
+                    this.state.googleFonts.push(font);
                 } else {
-                    this.googleLocalFonts.push(`'${font}': ''`);
+                    this.state.googleLocalFonts.push(`'${font}': ''`);
                 }
-                this.trigger_up('google_fonts_custo_request', {
-                    values: {[variable]: `'${font}'`},
-                    googleFonts: this.googleFonts,
-                    googleLocalFonts: this.googleLocalFonts,
+                this.state.option.instance._onGoogleFontsCustoRequest({
+                    // TODO: @owl-options should we allow something else that "variable" ? (most likely not)
+                    values: {[this.props.variable]: `'${font}'`},
```

Comment:
Not a huge fan of this, but OK for now. Let's just keep in mind that we should maybe have a way for widgets to request something from the option.